### PR TITLE
For #27029 - Use placeholder inplace of app name

### LINKED ISSUE #27029: Use placeholder inplace of app name for @string/wallpaper_classic_title
Coming from Use placeholder in place of app name for @string/wallpaper_classic_title"

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21593)

### PR DESCRIPTION
**Pull request by sarah541**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation
Fixes #27029

### CODE REVIEW

File: app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt [hunk 1]
```diff
@@ -116,9 +116,10 @@ private fun WallpaperGroupHeading(
     // and descriptions so they could be localized.
     // In the future, we may want to either use the dynamic wallpaper properties with localized fallbacks
     // or invest in a method of localizing the remote strings themselves.
-    if (collection.name == "classic-firefox") {
+    if (collection.name == "classic-firefox" || collection.name == "default") {
+        val appName = stringResource(R.string.app_name)
```
Comment (gabrielluong) [inline app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt:120]: Maybe let's just inline this?

**Review by gabrielluong — APPROVED** [review]
Can you update the "Use placeholder inplace of app name" commit message to include an issue number prior to landing? Let's say "For #26908 - Use placeholder inplace of app name for @string/wallpaper_classic_title" Thanks

**Comment by Mugurell** [comment]
I think this change would rather need a UX review.
Saw Delphine's comment from https://github.com/mozilla-l10n/android-l10n/pull/503#discussion_r971393633 but in this case "Classic Firefox" seems like a feature name on its own (also going by the `classic-firefox` collection name used in code). So it should not have a different name for Nightly/Beta/Release.

Seeing there "Classic Firefox Nightly" or "Classic Firefox Beta" seems unexpected to me.

**Comment by sarah541** [comment]
@Mugurell That's a valid point.

Waiting on UX and Product if they need this change

**Pull request merged**